PR: use attributes to address the missing article in the RHEL web console term (#3071)
theforeman/foreman-documentation: guides/common/modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc

[id="Managing_and_Monitoring_Hosts_Using_Cockpit_{context}"]

> = Managing and monitoring hosts using {CockpitL}

Author: ```suggestion
= Managing and monitoring hosts using {the-Cockpit}
```

You can access the {Cockpit} UI through the {ProjectWebUI} and use the functionality to manage and monitor hosts in {Project}.

.Prerequisites

> * {CockpitA} is enabled in {Project}.
> * {CockpitA} is installed on the host that you want to view:

Reviewer: ```suggestion
* You have enabled {CockpitL} in {Project}.
* You have installed {CockpitL} on the host that you want to view:
```

> * {CockpitA} is enabled in {Project}.
> * {CockpitA} is installed on the host that you want to view:

Author: ```suggestion
* You have enabled {the-Cockpit} in {Project}.
* You have installed {the-Cockpit} on the host that you want to view:
```

For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{context}[].

.Procedure

> . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host that you want to manage and monitor with {CockpitL}.

Author: ```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host that you want to manage and monitor with {the-Cockpit}.
```

. In the upper right of the host window, click the vertical ellipsis and select *Web Console*.

> You can now access the full range of features available for host monitoring and management, for example, {LoraxCompose}, through {CockpitL}.

Author: ```suggestion
You can now access the full range of features available for host monitoring and management, for example, {LoraxCompose}, through {the-Cockpit}.
```

ifdef::satellite[]
For more information about getting started with Red Hat web console, see {RHELDocsBaseURL}8/html/managing_systems_using_the_rhel_8_web_console/index[_Managing systems using the RHEL{nbsp}8 web console_] or {RHELDocsBaseURL}7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[_Managing systems using the RHEL 7 web console_].

> For more information about using {LoraxCompose} through {CockpitL}, see {RHELDocsBaseURL}8/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-web-console-interface_composing-a-customized-rhel-system-image#accessing-composer-gui-in-the-rhel-8-web-console_creating-system-images-with-composer-web-console-interface[_Accessing Image Builder GUI in the RHEL{nbsp}8 web console_] or {RHELDocsBaseURL}7/html/image_builder_guide/chap-documentation-image_builder-test_chapter_4#sect-Documentation-Image_Builder-Chapter4[_Accessing Image Builder GUI in the RHEL{nbsp}7 web console_].

Author: ```suggestion
For more information about using {LoraxCompose} through {the-Cockpit}, see {RHELDocsBaseURL}8/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-web-console-interface_composing-a-customized-rhel-system-image#accessing-composer-gui-in-the-rhel-8-web-console_creating-system-images-with-composer-web-console-interface[_Accessing Image Builder GUI in the RHEL{nbsp}8 web console_] or {RHELDocsBaseURL}7/html/image_builder_guide/chap-documentation-image_builder-test_chapter_4#sect-Documentation-Image_Builder-Chapter4[_Accessing Image Builder GUI in the RHEL{nbsp}7 web console_].
```
